src/Symfony/Component/Workflow/Definition.php

```diff
@@ -29,18 +30,28 @@ class Definition
      *
      * @param string[]     $places
      * @param Transition[] $transitions
+     * @param string       $initialPlace
      */
-    public function __construct(array $places = array(), array $transitions = array())
+    public function __construct(array $places = array(), array $transitions = array(), $initialPlace)
```

Comment:
I suppose that the default values for `$places` and `$transitions` can be removed, right?